components/SelectionInput.tsx

```diff
@@ -51,7 +51,7 @@ const SelectionInput = forwardRef<HTMLInputElement, Props>(
           }`}
         >
           <svg
-            className={`border rounded-full absolute h-5 w-5 p-0.5 ${
+            className={`border rounded absolute h-5 w-5 p-0.5 ${
```

Comment:
you should put the type condition here.
you have to add this condition also in `components/QuizExamForm.tsx`
```suggestion
            className={`border ${type==="checkbox"?"rounded":"rounded-full"} absolute h-5 w-5 p-0.5 ${
```